OIOUBL/src/SalesCreditMemo/COD13637.ExportSalesCreditMemo.al

```diff
@@ -303,14 +307,15 @@ codeunit 13637 "OIOUBL-Export Sales Cr. Memo"
             until SalesCrMemoLine2.NEXT() = 0;
         OIOUBLXMLGenerator.InsertLegalMonetaryTotal(XMLCurrNode, TaxableAmount, TaxAmount, TotalAmount, TotalInvDiscountAmount, CurrencyCode);
 
+        OnAfterAddCrMemoHeaderToDocument(XMLCurrNode, SalesCrMemoHeader);
```

Comment:
This is a odd location is it needed?

Reply:
In case I wan't to add extra elements or attributes to the header of the document.

Reply:
ok